SU-SWS/stanford_paragraph_types
# Earth-400 Responsive image paragraph type

# READY FOR REVIEW

# Summary
- Responsive image paragraph type

# Needed By (Date)
- 8/24

# Urgency
- mild

# Steps to Test

1. Checkout branch
2. enable `stanford_paragraph_responsive_image`
3. create complex page with responsive image.
4. verify it uses a responsive mode.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
